app/ui/src/assets/dictionary/en-GB.json

```diff
@@ -5,12 +5,40 @@
       "integration": "Integration",
       "integrations": "Integrations"
     },
+    "connections": {
+      "connection": "Connection",
+      "connections": "Connections"
+    },
     "dashboard": {
-      "header": "Dashboard",
+      "heading": "Dashboard",
       "metrics": {
         "sysmetrics": "System Metrics",
-        "uptime": "Uptime",
-        "integrationsheader": "{{0}} {{integrations.integrations}}"
+        "integrationsheader": "{{0}} {{integrations.integrations}}",
+        "totalmessages": "Total Messages",
+        "uptime": {
+          "uptime": "Uptime",
+          "since": "Since {{0}}"
+        },
+        "create": "{{create}} {{integrations.integration}}",
+        "viewall": "View All {{integrations.integrations}}"
+      },
+      "connections": {
+        "heading": "{{connections.connections}}",
+        "viewall": "View All {{connections.connections}}",
+        "create-btn-txt": "{{create}} {{connections.connection}}",
+        "empty-state-create-header": "{{create}} a {{connections.connection}}",
+        "empty-state-txt": "There are currently no connections available.  Please click on the button below to create one."
+      },
+      "integrations": {
+        "heading": "{{create}} an {{integrations.integration}}",
+        "empty-state-txt": "There are currently no {{integrations.integrations}} available. Please click on the button below to create one.",
+        "create-btn-txt": "{{create}} {{integrations.integration}}",
+        "top-integrations": {
+          "heading": "Top 5 {{integrations.integrations}}",
```

Comment:
What if we change the default recordset size to 6? We will want to track down all ocurrences in the dictionary then... 

Moving this to `shared.topitems` as `Top {{0}} {{1}}` would allow to customize the number of amounts displayed in the text and the item meant to be displayed in a more generic and reusable way.